src/index.ts

```diff
@@ -2,16 +2,18 @@ import cors from '@fastify/cors';
 import Fastify from 'fastify';
 import { PORT } from './config';
 import rootRouter from './routes';
-
+import dbConnector from './db/db';
 const fastify = Fastify();
-fastify.register(cors, {origin: true});
+fastify.register(cors, { origin: true });
 fastify.register(rootRouter);
+fastify.register(dbConnector);
 
 const start = async () => {
     try {
         await fastify.listen({ port: PORT });
         console.log(`server is listening on port ${PORT}`);
     } catch (err) {
+        console.log(err);
         fastify.log.error(err);
```

Comment:
wait why do we have two logs here? what is fastify.log.error? where does it go

Reply:
it came like this, I will delete it